src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -461,15 +461,15 @@ class TooltipHTMLContent {
 
     getSize() {
         const el = this.el;
-        return [el.offsetWidth, el.offsetHeight];
+        return el ? [el.offsetWidth, el.offsetHeight] : [0, 0];
     }
 
     moveTo(zrX: number, zrY: number) {
         const styleCoord = this._styleCoord;
         makeStyleCoord(styleCoord, this._zr, this._container, zrX, zrY);
 
         if (styleCoord[0] != null && styleCoord[1] != null) {
-            const style = this.el.style;
+            const style: any = this.el ? (this.el.style || {}) : {};
```

Comment:
Falling `el.style` back to `{}` makes no sense. It just circumvents those errors and causes unnecessary invoking. I would suggest:

```js
moveTo(zrX: number, zrY: number) {
  if (!this.el) {
    return;
  }
  // ...
}
```